lib/strategies/accept_header_strategy.rb

```diff
@@ -0,0 +1,42 @@
+module Phenotype
+  class AcceptHeaderStrategy
+    attr_reader :env, :content_type, :version_key
+
+    def initialize(app_name: 'rentpath',
+                   version_key: 'version', 
+                   format: 'json')
+      @version_key = version_key
+      @content_type = build_content_type(app_name, format)
+    end
+
+    def get_version(env)
```

Comment:
yes but my line of thinking was that this gems responsibility should just be choosing the correct rack app to call based on api version and nothing else, which it already does a great job at. 

at worst, if we needed hash representation of the ACCEPT header, we could easily write that up in the client.

is there a specific usecase you had in mind as far as how we could use the content negotiation stuff?